utilities/openemr-cmd/openemr-cmd

```diff
@@ -317,6 +323,7 @@ then
   echo "  start                              Execute: docker-compose start"
   echo "  stop                               Execute: docker-compose stop"
   echo "  s, shell                           Open a docker shell quickly"
+  echo "  ms, maria-shell                    Open a maria docker shell quickly"
```

Comment:
would qualify that this won't work in the insane dev (since there are lots of mysql/mariadb dockers :) ). something like:
```
Open a maria docker shell quickly (only works in the easy dev environment)
```